packages/core/src/external/carequality/ihe-gateway-v2/ihe-gateway-v2-logic.ts

```diff
@@ -128,7 +162,9 @@ export async function createSignSendProcessXCPDRequest({
       log(`${msg} - ${errorToString(error)} - ${JSON.stringify(extra)}`);
       capture.error(msg, { extra: { ...extra, error } });
     }
-  }
+  });
+
+  await Promise.allSettled(resultPromises);
```

Comment:
also not sure we need to await / settle these right?

Reply:
we definetly need to. There execution will be stopped prematurely right re what Raf said above about the lambda env killing node